packaging/bin/pki-enroll-openssh-cert.sh

```diff
@@ -32,6 +32,7 @@ sign() {
 		}
 		trap cleanup 0
 		cat "${PKIDIR}/private/ca.pem" > "${TMPCA}"
+        umask "$(/bin/sh -l -c umask)"
```

Comment:
If there is no umask set explicitly in shell login scripts (/etc/profile, ...) then the umask is inherited from the calling environment. It shouldn't be a problem in RHEL-based systems though.